scripts/asr_language_modeling/ngram_lm/ngram_merge.py

```diff
@@ -336,12 +327,15 @@ def make_symbol_list(nemo_model_file, symbols, force):
     else:
         if nemo_model_file.endswith('.nemo'):
             asr_model = nemo_asr.models.ASRModel.restore_from(nemo_model_file, map_location=torch.device('cpu'))
-            vocab_size = len(asr_model.decoder.vocabulary)
         else:
             logging.warning(
                 "nemo_model_file does not end with .nemo, therefore trying to load a pretrained model with this name."
             )
             asr_model = nemo_asr.models.ASRModel.from_pretrained(nemo_model_file, map_location=torch.device('cpu'))
+
+        if isinstance(asr_model.decoder, RNNTDecoder):
```

Comment:
@titu1994 is it the best way to have the vocab size?

Reply:
As long as its not a TDT model or a Multiblank RNNT model I think. Otherwise you gotta add another value that counts the number of blank/duration ids. But this code is fine for RNNT alone until we support Beam search with TDT